docker/test/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  db:
+    ports:
+      - 5433:5432
```

Comment:
I know it´s strange but it seems there is bug with mapping ports with the same number so I had to do this.